mockgcp/mockaccesscontextmanager/service.go

```diff
@@ -74,7 +74,7 @@ func (s *MockService) NewHTTPMux(ctx context.Context, conn *grpc.ClientConn) (ht
 		return nil, fmt.Errorf("error building grpc service: %w", err)
 	}
 
-	grpcMux.AddService(pb.NewAccessContextManagerClient(conn))
+	grpcMux.AddService(pb.NewAccessContextManagerClient(conn), httptogrpc.WithServiceName("google.identity.accesscontextmanager.v1.AccessContextManager"))
```

Comment:
Good usage of the new option. This bypasses the potentially buggy `findServiceName` logic for this specific service.